recipe/meta.yaml

```diff
@@ -8,9 +8,8 @@ package:
   version: {{ version }}
 
 source:
-  # the tool is GPL licensed but the download is not possible without registration, so we use a copy here
-  url: https://bitbucket.org/simonbray/ambertools/raw/9e3a9289ca97a2ba0f58cebdfb102c88e8912d9f/AmberTools19.tar.bz2
-  sha256: 8315929b8e1d597907405ddab31d41c556a777f7df7df98745ee9d08c980bae5
+  url: https://ambermd.org/downloads/AmberTools19.tar.bz2
+  sha256: 0c86937904854b64e4831e047851f504ec45b42e593db4ded92c1bee5973e699
 
 build:
   number: 0
```

Comment:
@jaimergp, should we bump this?

Reply:
Now that we might need to change the requirements, yes! Bumped up!